setup.py

```diff
@@ -47,7 +47,7 @@ def check_dependencies():
                             'clean'))):
```

Comment:
I would add a message to the user at the end of install about where the executable is placed. For example, I installed using an anaconda2 python environment, and despite the bin not being on my path, my naive expectation was to type "which pydeface [TAB]" and find something. I didn't (obviously) but if the installer had told me, I might have not been this silly :) Another suggestion (which is just your preference) is to specify a particular entrypoint and then make the script look more like an executable (versus a python script you call) and (there are multiple ways to go about this) but I would move the executable from scripts into the package (call it main.py or the like) and then in the setup:

```
entry_points = {'console_scripts': [ 'deface=pydeface.main:main' ] }
```